plugins/github/lib/samson_github/samson_plugin.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 module SamsonGithub
-  STATUS_URL = ENV["GITHUB_STATUS_URL"] || 'https://status.github.com'
+  # Reference: https://www.githubstatus.com/api#status
+  # Official doc example is: https://kctbh9vrtdwd.statuspage.io',
+  # however 'https://www.githubstatus.com' also works
+  STATUS_URL = ENV["GITHUB_STATUS_URL"] || 'https://www.githubstatus.com'
```

Comment:
do we know what ENV["GITHUB_STATUS_URL"] value is initialized with?

Reply:
That can be set during deploy time, for example chef or your script runner to start the server. Nevertheless, the default behaviour still needs to get fixed.